### PR TITLE
Create ci.yml

### PR DESCRIPTION
**Pull request by LindseySaari**
The ci.yml will create a semantic release following push to the master branch. Additionally, upon push to master, the image tag in the `vsp-infra-application-manifests` repo will be updated to the latest ECR tag. 

**Important:** In order for the semantic release to be successful, developers will have to use the semantic release syntax via your commit message when committing to the master branch. See [this doc ](https://github.com/semantic-release/semantic-release#how-does-it-work)

### CODE REVIEW

**Comment by thilton-oddball** [comment]
As extensive as it looks like the semantic release syntax is, I wonder if we should include some specific examples in our README. Thoughts?

**Comment by LindseySaari** [comment]
> As extensive as it looks like the semantic release syntax is, I wonder if we should include some specific examples in our README. Thoughts?

Great idea! I'm going to update the readme!

**Review by thilton-oddball — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+name: Release and Update Manifests
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      environments:
+        description: "Provide a space delimited list of the cluster environments you would like to update wrapped in single quotes ex. 'dev' or 'dev staging sandbox'.  Avalable options -> dev, staging, sandbox, utility, prod sandbox"
```
Comment (ericboehs) [inline .github/workflows/ci.yml:10]: Can we wrap or shorten this line?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      - name: Add and Commit file
+        if: steps.semantic.outputs.new_release_published == 'true' || github.event.inputs.environments != ''
+        uses: EndBug/add-and-commit@v7
+        with:
+          branch: main
+          add: '*'
+          cwd: vsp-infra-application-manifests/apps/vsp-tools-backend/platform-console
+          author_name: va-vsp-bot
+          author_email: devops@va.gov
+          message: 'auto update platform-console ${{ needs.prepare-values.outputs.environments }} images'
```
Comment (ericboehs) [inline .github/workflows/ci.yml:104]: Would it make sense to add the commit range or release version in this message?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,104 @@
+name: Release and Update Manifests
```
Comment (ericboehs) [inline .github/workflows/ci.yml:1]: ci.yml seems like the wrong name? Maybe release_and_publish or trigger_deploy.yml, or maybe just deploy.yml?

**Review by ericboehs — COMMENTED** [review]
Do you have any thoughts on how we could clean this up for VFS apps in the future to use? Maybe we could extra a GH action that they could import and pass a few variables to?

File: .github/workflows/ci.yml (hunk 2 above)
Comment (LindseySaari) [reply .github/workflows/ci.yml:104]: > Do you have any thoughts on how we could clean this up for VFS apps in the future to use? Maybe we could extra a GH action that they could import and pass a few variables to?

Yes good idea... I believe they would only need the cluster env, and a few dir names from the manifests repo that they could pass in

**Comment by LindseySaari** [comment]
> Do you have any thoughts on how we could clean this up for VFS apps in the future to use? Maybe we could extra a GH action that they could import and pass a few variables to?

Yes good idea... I believe they would only need the cluster env, and a few dir names from the manifests repo that they could pass in

**Pull request merged**